assets/dev/js/frontend/handlers/base-tabs.js

```diff
@@ -162,7 +162,7 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 						this.handleKeyboardNavigation( event );
 						break;
 					case 'Enter':
-					case 'Space':
+					case ' ': // Space.
```

Comment:
The `event.code` is `Space`, switch to check the `code` instead of the `key`

Reply:
Handled.